src/masternode-budget.cpp

```diff
@@ -179,11 +179,11 @@ void CBudgetManager::SubmitFinalBudget()
     std::string strBudgetName = "main";
     std::vector<CTxBudgetPayment> vecTxBudgetPayments;
 
-    for (unsigned int i = 0; i < vBudgetProposals.size(); i++) {
+    for (auto & vBudgetProposal : vBudgetProposals) {
```

Comment:
Can make this
```cpp
for (const auto& vBudgetProposal: vBudgetProposals) {
```
since we are at it (only getters are accessed, which are `const`).